docker/.env

```diff
@@ -0,0 +1,2 @@
+TAG=TEST.17.1
+COMPOSE_PROJECT_NAME=massa
```

Comment:
Those kind of information should not be hardcoded but passed by the continuous integration through build arguments.

Reply:
The TAG parameter changes dynamically when executing docker/build.sh
if [[ "$(uname)" == "Darwin" ]]
then
    sed -i "" "s/^TAG=.*$/TAG=${IMAGE_TAG}/" "$DIR/.env"
else
    sed -i "s/^TAG=.*$/TAG=${IMAGE_TAG}/" "$DIR/.env"
fi